services/checks/checks.go

```diff
@@ -532,7 +604,12 @@ func hashID(s string) string {
 	return hex.EncodeToString(data[:])
 }
 
-func (s *Service) createAlert(id, name string, target target, result *check.Result) {
+type alertWithID struct {
+	alert ammodels.PostableAlert
+	id    string
+}
+
+func (s *Service) createAlert(name string, target target, result *check.Result) alertWithID {
```

Comment:
maybe it's better to rename this method to `prepareAlert` or something like that?

Reply:
hmm.. I'm okay with either of those names and personally I don't find `createAlert` to be ambiguous enough to rename it, but if you feel otherwise I am okay with renaming the method